cabot/cabotapp/tests/tests_basic.py

```diff
@@ -231,8 +260,8 @@ def test_timeout_handling_in_jenkins(self):
         self.jenkins_check.run()
         checkresults = self.jenkins_check.statuscheckresult_set.all()
         self.assertEqual(len(checkresults), 1)
-        self.assertTrue(self.jenkins_check.last_result().succeeded)
-        self.assertIn(u'Error fetching from Jenkins - фиктивная ошибка',
+        self.assertFalse(self.jenkins_check.last_result().succeeded)
+        self.assertIn(u'Error fetching from Jenkins - why was this in russian',
```

Comment:
Yeah :( I took it out because the version of cabot we're using right now doesn't seem to support non-ascii characters--when i first ran the tests with no changes I got
`'ascii' codec can't encode characters in position 0-8: ordinal not in range(128)`

I'll fix it now